dedicated-community-leader.md

```diff
@@ -0,0 +1,81 @@
+- The initially selected developer community leader does not have an extensive network within the company.
+
+**Review Comments**
+- (**Done**) clarify who actually bought in to which extent
+- (**Done**) clarify how many resources are available to the InnerSource effort (marketing, capacity)
+- (**Done**) add to first context bullet point: traditional companies might not have the right culture for communities to form naturally without much external stimulation and support.
+- (**Done**) clarify size of company
+
+## Forces
+
+If a company does not significantly invest in the initial InnerSource community in terms of budget and capacity for InnerSource, the credibility of its commitment to InnerSource might be perceived as questionable. A common impulse of a company with a traditional management culture to a project or initiative not performing as expected will be to replace its leader. Doing that without involving the community and following meritocratic principles will further undermine the companies commitment to InnerSource by highlighting the friction between the current company culture and the target culture - a community culture.
+
+The value contribution of InnerSource projects will not be obvious for many managers which are steeped in traditional project management methods. Those managers are less likely to assign one of their top people, who are usually in high demand by non InnerSource-projects, to an InnerSource project for a significant percentage of their work time.
+
+Community work - especially communication - make up for a significant percentage of a community leaders daily work. At the same time, he will likely also have to spearhead the initial development, too. In the face of limited capacity, inexperienced leaders will tend to focus on development and neglect communication. The barrier for potential contributors to make their first contribution and to commit to the community will be much higher if the community leader is hard to reach or is slow to respond to feedback and questions for lack of time. Furthermore, technically inexperienced leaders will most likely have a harder time to attract and retain highly experienced contributors than a top performer with a high degree of visibility within a company would have. 
+
+If a community can not grow fast enough and pick up enough speed, chances are they won't be able to convincingly demonstrate the potential of InnerSource.
+
+**Review comments**
+- (**Open**: I have added something to the problem statement instead) now added that to the maybe reference pattern tbd (start small, experiment then scale up as it has proven successful)
+
+## Solution
+
+Select a community leader who:
+- is experienced in the Open Source working model or similar community based working models, 
+- has the required soft-skills to act as a natural leader,
+- is an excellent networker and who
+- inspires community members.
+
```

Comment:
Management needs to be sensitive to the needs of the open community before they engender a change in community manager.